### PR TITLE
Added Hint.toCredentialBuilder() & Hint.Builder(Hint)

### PR DESCRIPTION
**Pull request by dxslly**
- Added Hint.toCredentialBuilder() & Hint.Builder(Hint)
- Merged TestFixtures into TestConstants

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -4,3 +4,4 @@ distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+org.gradle.jvmargs=
```
Comment (jstimes) [inline gradle/wrapper/gradle-wrapper.properties]: is this intentional or did you forget a value?

File: protocol/javatests/org/openyolo/protocol/CredentialTest.java [hunk 2]
```diff
@@ -39,10 +39,17 @@
     public static final AuthenticationDomain AUTH_DOMAIN =
             new AuthenticationDomain(AUTH_DOMAIN_STR);
 
+    public static final String ID_TOKEN =
```
Comment (jstimes) [inline protocol/javatests/org/openyolo/protocol/CredentialTest.java]: Does this need to be public?

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (dxslly) [reply gradle/wrapper/gradle-wrapper.properties]: Thanks, unintentional change.

File: protocol/javatests/org/openyolo/protocol/CredentialTest.java (hunk 2 above)
Comment (dxslly) [reply protocol/javatests/org/openyolo/protocol/CredentialTest.java]: Fixed.

**Pull request merged**